scene/2d/physics_body_2d.cpp

```diff
@@ -362,12 +363,12 @@ struct _RigidBody2DInOut {
 	int local_shape;
 };
 
-bool RigidBody2D::_test_motion(const Vector2 &p_motion, float p_margin, const Ref<Physics2DTestMotionResult> &p_result) {
+bool RigidBody2D::_test_motion(const Vector2 &p_motion, bool p_infinite_inertia, float p_margin, const Ref<Physics2DTestMotionResult> &p_result) {
```

Comment:
You forgot to update the `RigidBody.test_motion()` bindings here https://github.com/godotengine/godot/blob/master/scene/2d/physics_body_2d.cpp#L890 to add the infinite_inertia parameter

Reply:
Oh thanks you... I forgot it because yesterday I got some issue with bindings so I removed it from all bindings to test it and apparently I forgot to re-add it.

Thanks you